src/app/container/register-tool/register-tool.service.ts

```diff
@@ -26,6 +26,8 @@ import { MetadataService } from './../../shared/swagger/api/metadata.service';
 import { DockstoreTool } from './../../shared/swagger/model/dockstoreTool';
 import { Tool } from './tool';
 import { HostedService } from './../../shared/swagger/api/hosted.service';
+// This line is super important for jQuery to work across the website for some reason
+import * as $ from 'jquery';
```

Comment:
Would `import * as jQuery from 'jquery';` work to avoid some of the changes below?

Reply:
we had $ in some places and jQuery in some other places.  I decided it would be best to keep things consistent.  I pretty much randomly chose $ over jQuery.